packages/o-spreadsheet-engine/src/xlsx/functions/charts.ts

```diff
@@ -992,23 +1011,28 @@ function addDoughnutChart(
       <c:holeSize val="${
         chart.pieHolePercentage ?? (chart.isDoughnut ? DEFAULT_DOUGHNUT_CHART_HOLE_SIZE : 0)
       }" />
-      ${insertDataLabels()}
       ${joinXmlNodes(dataSetsNodes)}
+      ${insertDataLabels({ showValues: chart.showValues })}
     </c:doughnutChart>
   `;
 }
 
-function insertDataLabels({ showLeaderLines } = { showLeaderLines: false }): XMLString {
+function insertDataLabels(
```

Comment:
I know it was done this way before but I'm not sure we should pass and object here for only two arguments